astrogrism/config/HST/create_HST_reffiles.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+from astrogrism.config.HST.reference_file_generators._generate_specwcs import create_grism_specwcs  # noqa
+from astrogrism.config.HST.reference_file_generators._generate_wavelengthrange import create_tsgrism_wavelengthrange  # noqa
+# from astrogrism.config.HST.reference_file_generators.reference_file_generators._generate_distortion import create_distortion  # noqa
+
+
+def create_reference_files(conffile, hst_grism, outpath=Path.cwd(), outbasename="HST"):
+    create_tsgrism_wavelengthrange(outname=str(Path(outpath) / (
+                                                                str(outbasename) +
+                                                                "_wavelengthrange.asdf")
+                                               )
+                                   )
+    create_grism_specwcs(conffile=str(conffile),
+                         pupil=hst_grism,
+                         outname=str(Path(outpath) / (str(outbasename) + "_wavelengthrange.asdf")))
```

Comment:
```suggestion
                         outname=str(Path(outpath) / (str(outbasename) + "_specwcs.asdf")))
```

Additionally, how about rather than the default basename being "HST", just make it `None` and strip the extension off the .CONF file for the basename in that case? Then any information there would be preserved. I'm basically thinking that whoever we pass this off to will want to be as lazy as possible in running it.

Reply:
Thanks for catching the typo. And yes, that also sounds like a good suggestion (wasn't sure which of the two suggestions you were leaning towards in https://github.com/spacetelescope/astrogrism/pull/12#discussion_r763020203

How does this look?

Reply:
Better! I left two more minor comments, then hopefully we can merge this. I'll do another quick pass over the rest of it.